tests/Samples.Server.Tests/ShouldBeExtensions.cs

```diff
@@ -29,7 +29,21 @@ public static void ShouldBe(this string actual, string expected, bool ignoreExte
                 }
             }
 
-            actual.ShouldBe(expected);
+            actual.ShouldBeEquivalentJson(expected);
+        }
+
+        /// <summary>
+        /// Compares two strings but replaces any equivalent encoding differences first.
+        /// </summary>
+        /// <param name="actualJson"></param>
+        /// <param name="expectedJson"></param>
+        public static void ShouldBeEquivalentJson(this string actualJson, string expectedJson)
+        {
+#if !NETCOREAPP2_2 // e.g. System.Text.Json is being used which encodes slightly differently
+            expectedJson = expectedJson.Replace("'", @"\u0027");
```

Comment:
Wouldn’t it be easier to remove the preprocessor directive and always call replace on both parameters?

Reply:
Good call. Done.